cmd/collectors/zapiperf/plugins/volume/volume.go

```diff
@@ -32,51 +33,65 @@ func (me *Volume) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 	opsKeyPrefix := "temp_"
 	re := regexp.MustCompile(`^(.*)__(\d{4})$`)
 
-	metricName := "aggrs"
-	metric, err := data.NewMetricFloat64(metricName, metricName)
+	flexgroupAggrsMap := make(map[string]*set.Set)
+	// new metric would be volume_aggr_labels
+	metricName := "aggr_labels"
+	volumeAggrmetric := matrix.New(".Volume", "volume", "volume")
+	volumeAggrmetric.SetGlobalLabels(data.GetGlobalLabels())
+
+	metric, err := volumeAggrmetric.NewMetricFloat64(metricName)
 	if err != nil {
 		me.Logger.Error().Stack().Err(err).Msg("add metric")
```

Comment:
`Stack()` not needed for any of the log msgs in this class. The msg and file:line is sufficient for root cause and troubleshooting

Reply:
done